root/email_templates/generic/session_resent_body.mhtml

```diff
@@ -6,48 +6,40 @@ $session_link
 $curator_name
 $help_index
 $existing_annotation_count
+$logged_in_user
 </%args>
 Dear <% $curator_name %>,
 
-Congratulations on the recent publication of your paper:
-  <% $publication_uniquename %>
-<% wrap('  ', '  ', '"' . $publication_title . '"') %>
+We recently invited you to contribute annotations based on your paper,
+<% $publication_uniquename %> -
+<% wrap('  ', '  ', '"' . $publication_title . '"') %>,
+to <% $config->{database_name} %>.
 
-We invite you to contribute annotations based on this paper. We have
-identified the paper as suitable for curation in Canto, but if it
-reports data for a large number of genes, please let us know so we can
-collect the data by a different route.
+We have noticed, however, that no action has been taken on the curation
+session. Please visit this page, where you can either start curating the
+paper, or reassign it to another author (e.g. the first author).
 
-Your curation link is:
```

Comment:
Can we keep this line or something like it?  I'd like to be clear what the link is for.

Reply:
I'll probably have to reword the previous paragraph as well but yes, I'll make that change.

Reply:
Fixed by https://github.com/pombase/canto/pull/2501/commits/ee018c5f2c13cea36dc78462797210042cfa1114.